modules/tinymce/src/themes/silver/main/ts/ui/dialog/SizeInput.ts

```diff
@@ -34,8 +34,7 @@ export const renderSizeInput = (spec: SizeInputSpec, providersBackstage: UiFacto
       classes: [ 'tox-lock', 'tox-button', 'tox-button--naked', 'tox-button--icon' ],
       attributes: {
         'aria-label': translatedLabel,
-        // TINY-10453: Remove this tooltip, we don't want duplicate tooltips and until we figured a better way to test, it's here now so that tests would pass
-        'title': translatedLabel
+        'data-mce-btn': spec.label.getOr('Constrain proportions')
```

Comment:
Probably worth having a label variable like `const label = spec.label.getOr('Constrain proportions')`; to avoid the duplication

Reply:
Fixed 👍  https://github.com/tinymce/tinymce/pull/9312/commits/61bd86bad49010bf1144e64bf6b967b19e5ed11b